### ISSUE TITLE
Clarify formatting of new lines?

### ISSUE THREAD

**Issue by NY00123** [FrenkelS/keensource456tandy#17]
I got unexpected compilation errors, until I converted newlines to CR+LF. The files in the git repository seem to use LF format. Maybe this should be documented in the readme, if the repository itself isn't updated accordingly.

**Comment by FrenkelS** [comment]
I've added `.gitattributes` because https://www.aleksandrhovhannisyan.com/blog/crlf-vs-lf-normalizing-line-endings-in-git/ says I should. Does this solve the problem?

**Comment by NY00123** [comment]
Thanks for having a look. I made a new clone, and it looks like the files still use just line feeds for the new lines.

I should had probably written earlier that the clones were made from a Linux distribution, so LF chars probably remained as-is by the process.

While modern editors (like multiple versions of Visual Studio) seem to support all formats, relevant Borland versions are apparently restricted to CR+LF.

**Comment by FrenkelS** [comment]
I've converted all the newlines to LF on my Windows computer and when I compile it in Borland C++ v3.0 in DOSBox I get a few errors. The errors disappear when I convert the newlines to CR+LF in just 4 files: CK_DEF.H, ID_CA.C, ID_VW.C and ID_VW.H.

I've also tried to convert all the newlines to CR. I get the same errors, plus now all the .ASM and .EQU files result in errors.

**Comment by FrenkelS** [comment]
I've added *.C, *.H, *.ASM and *.EQU to .gitattributes.
That should fix it.

**Comment by NY00123** [comment]
Thanks for the fix! I made a new git clone, and following the assistance of command-line tools, I confirm that the sources use CR-LF now. Only makeobj.c isn't CR-LF formatted (due to having a lowercase file extension), but MAKEOBJ.EXE already exists.

I further confirm that I could build executables after creating the "OBJ" subdirs.

I couldn't build KEEN4E.EXE because RFL_InitAnimList was changed to a static function, but I otherwise got all exes to be successfully built.
